_posts/2022-04-25-announcing-call-for-proposals.md

```diff
@@ -0,0 +1,30 @@
+---
+author: DjangoCon Organizers
+category: General
+date: 2022-04-25 12:00:00
+layout: post
+image: /static/img/blog/cfp-announced-2022.png
+post_photo_url: /static/img/blog/speaking-2022-cfp.jpg
+post_photo_alt: "Speaker addressing a crowd at DjangoCon US 2019 in San Diego"
+title: "The Call for Proposals for DjangoCon US 2022 Is Now Open!"
+---
+
+The DjangoCon 2022 organizers are excited to announce that the first in-person DjangoCon since 2019 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2022/cfp)! The deadline for submissions is May 31th, 2022 [AoE](https://time.is/compare/0000_31_May_2022_in_Anywhere_on_Earth). As long as it’s still May 31th anywhere on earth, you can submit your proposal.
```

Comment:
```suggestion
The DjangoCon 2022 organizers are excited to announce that the first in-person DjangoCon since 2019 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2022/cfp)! The deadline for submissions is May 31st, 2022 [AoE](https://time.is/compare/0000_31_May_2022_in_Anywhere_on_Earth). As long as it’s still May 31st anywhere on earth, you can submit your proposal.
```

Reply:
Looks good to me.